helm/prometheus-meta-operator/templates/configmap.yaml

```diff
@@ -80,5 +80,10 @@ data:
         region: onprem
         {{- end }}
         registry: {{ .Values.registry.domain }}
+        {{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1/Ingress" }}
```

Comment:
for my understanding - could we query the api server for this, instead of having it configurable?

Reply:
Yes helm checks that using the kubernetes API. If we were to add that to our codebase, this would basically mean importing helm code in pmo